wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewModel.swift

```diff
@@ -232,6 +235,28 @@ final class ConversationGuestOptionsViewModel {
         }
     }
 
+    /// Creates a Guest Link
+    /// - Parameter view: the source view which triggers create action
+    /// - Returns: alert controller
+    @discardableResult
+    func showAlertForGuestLink(from view: UIView? = nil) -> UIAlertController? {
```

Comment:
The method name is a bit misleading since it won't always show an alert. I suggestion something like `startGuestLinkCreationFlow`

Reply:
Same with this one!